src/Authentication/Authentication.Core/Interfaces/IGraphRequestProofofPossession.cs

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Azure.Core;
+using Azure.Identity;
+using System;
+using System.Net.Http;
+
+namespace Microsoft.Graph.PowerShell.Authentication
+{
+    public interface IGraphRequestProofofPossession
```

Comment:
Prefer to use objects instead of interfaces. You will not be able to add members here.

Also there is no reason why this is an interface. A data object with public setters is perfectly testable.